### PR TITLE
Improved accessibility of player interfaces

### LINKED ISSUE #4303: [Unified] Accessibility: most buttons have no title
### Version

- b69e477

### Steps to reproduce the bug
1. Enable "text to speech" in accessibility settings
2. open a video in NewPipe
3. Play a video
4. tap on any button in the video ->  

### Expected behavior
You'll get a description of the button's action.

### Actual behaviour
Device says "unlabelled button"

### LINKED ISSUE #10196: The app's player page does not have a Chinese accessibility tab.
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

It is hoped that the player page of the application and the playback control bar below will be added with a Chinese accessibility label.

### Why do you want this feature?

I am blind and cannot operate the player page properly when using the app.

### Additional information

Accessibility related content is listed in Google's Accessibility Development Guide.

### PR DESCRIPTION
**Pull request by TobiGr**
#### What is it?
- [x] Bugfix (user facing)
- [ ] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)

#### Description of the changes in your PR
Add content descriptions for most player actions.

#### ToDo
- [ ] ~~Test the changes and wait for feedback~~ No feedback from original reporters

#### Fixes the following issue(s)
- Fixes #4303
- Fixes #10196 

#### APK testing
Download the APK via this URL: https://github.com/TeamNewPipe/NewPipe/suites/13997135622/artifacts/780985251
This downloads a zip file (`app.zip`) which contains the debug APK of this PR.

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -790,4 +790,15 @@
...
+    <string name="toggle_fullscreen">toggle fullscreen</string>
+    <string name="toggle_screen_orientation">toggle screen orientation</string>
+    <string name="previous_stream">previous stream</string>
+    <string name="next_stream">next stream</string>
+    <string name="play">play</string>
+    <string name="replay">replay</string>
+    <string name="more_options">more options</string>
+    <string name="duration">duration</string>
+    <string name="rewind">rewind</string>
+    <string name="forward">forward</string>
```
Comment (Stypox) [inline app/src/main/res/values/strings.xml:803]: Why are they all lowercase?

**Review by Stypox — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (TobiGr) [reply app/src/main/res/values/strings.xml:803]: Should they be uppercase? If yes, why?

Comment (Stypox) [reply app/src/main/res/values/strings.xml:803]: The first letter should be uppercase, shouldn't it?

**Review by Stypox — APPROVED** [review]
Thanks!

**Comment by TobiGr** [comment]
rebased

**Pull request merged**